modules/monitoring-default-monitoring-components.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * monitoring/understanding-the-monitoring-stack.adoc
+
+[id="default-monitoring-components_{context}"]
+= Default monitoring components
+
+By default, the {product-title} {product-version} monitoring stack includes these components:
+
+.Default monitoring stack components
+[options="header"]
+|===
+
+|Component|Description
+
+|Cluster Monitoring Operator
+|The Cluster Monitoring Operator (CMO) is a central component of the monitoring stack. It deploys and manages Prometheus instances, the Thanos Querier, the Telemeter Client, and metrics targets and ensures that they are up to date. The CMO is deployed by the Cluster Version Operator (CVO).
+
+|Prometheus Operator
+|The Prometheus Operator (PO) in the `openshift-monitoring` project creates, configures, and manages platform Prometheus instances and Alertmanager instances. It also automatically generates monitoring target configurations based on Kubernetes label queries.
+
+|Prometheus
+|Prometheus is the monitoring system on which the {product-title} monitoring stack is based. Prometheus is a time-series database and a rule evaluation engine for metrics. Prometheus sends alerts to Alertmanager for processing.
+
+|Prometheus Adapter
+|The Prometheus Adapter (PA in the preceding diagram) translates Kubernetes node and Pod queries for use in Prometheus. The resource metrics that are translated include CPU and memory utilization metrics. The Prometheus Adapter exposes the cluster resource metrics API for horizontal Pod autoscaling. The Prometheus Adapter is also used by the `oc adm top nodes` command.
+
+|Alertmanager
+|The Alertmanager service handles alerts received from Prometheus. Alertmanager is also responsible for sending the alerts to external notification systems.
+
+|`kube-state-metrics`
```

Comment:
should delete the `` , then it uses the same font as others
`kube-state-metrics` is fine, it is the same with `openshift-state-metrics` and `node-exporter`

Reply:
Thank you for your suggestion. Following a review on this, I have decided to keep the back ticks in place because this refers to the Pod name and agent name. To help with the styling, I have added the word agent for each. For example:`kube-state-metrics` agent.